stable/home-assistant/values.yaml

```diff
@@ -102,6 +102,8 @@ zwave:
   enabled: false
   device: ttyACM0
 
+deviceMounts: []
```

Comment:
Shouldn’t this be ‘hostMounts’ ?

Reply:
Yikes, good catch. I changed approaches at the last minute and decided to expand it to support all host mounting, not just `/dev/`, and forgot to test with the default config. Updated now, and also added some comments in the `values.yaml` for sample usage.